libs/platform/table/components/table-p13-dialog/table-p13-dialog.component.ts

```diff
@@ -78,6 +86,10 @@ export class TableP13DialogComponent implements OnDestroy {
         return this._table;
     }
 
+    /** Event emitted when dialog is closed. */
+    @Output()
+    closeDialog = new EventEmitter<string[]>();
```

Comment:
In my opinion, should be called `dialogClosed`, in past-tense, since it is an event that is triggered after closing. `closeDialog` gives me the impression it's a method that can be called that will then close the dialog

Reply:
Make sense, I've changed it.